contracts/staking/DefaultTargetDispenserL2.sol

```diff
@@ -140,7 +140,7 @@ abstract contract DefaultTargetDispenserL2 is IBridgeErrors {
 
     /// @dev Processes the data received from L1.
     /// @param data Bytes message data sent from L1.
-    function _processData(bytes memory data) internal {
+    function _processData(bytes memory data) internal returns (uint256 totalAmount) {
```

Comment:
Since proposed deposited amounts could be further cut off by the StakingFactory verification and result in more withheldAmount-s, let's compute the total amount that is deposited.